components/Reviews/components/RatingNumber/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from '@shopgate/pwa-common/components/I18n';
+import { RATING_SCALE_DIVISOR } from '@shopgate/pwa-ui-shared/RatingStars/constants';
+
+/**
+ * Rating number component.
+ * @param {Object} props Props.
+ * @param {number} props.rating Rating value.
+ * @param {string} props.className Class name.
+ * @returns {JSX}
+ */
+const RatingNumber = ({ rating, className }) => {
+  if (!rating) {
```

Comment:
what if `rating === 0` ?

Reply:
It should not happen but you're right, it's technically possible. Changed the code.